lib/services/services_lsb.c

```diff
@@ -52,15 +52,15 @@
                            " " PCMK_META_TIMEOUT "='15s'"                     \
                            " " PCMK_META_INTERVAL "='15s' />\n"               \
     "  </" PCMK_XE_ACTIONS ">\n"                                              \
-    "  <special tag='LSB'>\n"                                                 \
+    "  <" PCMK_XE_SPECIAL " tag='LSB'>\n"                                     \
     "    <Provides>%s</Provides>\n"                                           \
```

Comment:
It's not clear to me why we need any of these special attributes, in `st_lha.c`, `services_lsb.c`, `systemd.d`, or `upstart.c`. "special" is reserved for things not specifically part of the OCF spec. What's using these?

Reply:
Just the user. It's just a way to not lose useful information from the init script header.